frontend/src/lib/components/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,64 @@
+import clsx from 'clsx'
+import { range } from 'lib/utils'
+import React from 'react'
+import { LemonButtonProps } from '../LemonButton'
+import './Skeleton.scss'
+
+export interface SkeletonProps {
+    className?: string
+    width?: string | number
+    height?: string | number
+    repeat?: number
+    active?: boolean
+}
+
+export function Skeleton({ className, width, height, repeat, active = true }: SkeletonProps): JSX.Element {
```

Comment:
Can we add the old Ant `Skeleton` to "soft-prohibited" components, like `Button`? (Not sure if this can work without renaming to `LemonSkeleton`)

Reply:
I couldn't find a straightforward way of doing this without renaming to LemonSkeleton... Maybe we just do rename it for now?

Reply:
I think `LemonSkeleton` would be widely understood

Reply:
Went for the rename 👍